tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1910,57 +1911,14 @@ public synchronized Map<Integer, VoterInfo> getVoterToInfo() {
      */
     public static String[] getOrgList() {
         if (orgList.length == 0) {
-            resetOrgList();
-        }
-        return orgList;
-    }
-
-    private static void resetOrgList() {
-        // get all orgs in use...
-        Set<String> orgs = new TreeSet<>();
-        Connection conn = null;
-        Statement s = null;
-        try {
-            conn = DBUtils.getInstance().getAConnection();
-            s = conn.createStatement();
-            ResultSet rs = s.executeQuery("SELECT distinct org FROM " + CLDR_USERS + " order by org");
-            while (rs.next()) {
-                String org = rs.getString(1);
-                orgs.add(org);
+            // Get the preferred "display" names according to Organization.java
+            Set<String> names = new TreeSet<>();
+            for (Organization o: Organization.values()) {
+                names.add(o.name());
```

Comment:
no need to read the db to get the org list -- this becomes much simpler

Reply:
Good change — so much for https://github.com/unicode-org/cldr/pull/2032 merging anytime soon!